src/context/TodoContext.tsx

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import React, { useMemo } from 'react';
+
+import { useLocaleStorage } from '../hooks/useLocalStorage';
```

Comment:
The imported hook useLocaleStorage might be a typo, it should be useLocalStorage.